GPS next_WP #
GPS cur_pos #

float64 next_vel #
float64 cur_vel #

float64 next_heading #
float64 cur_heading #

int64 time
int64 gps_time #

float64 battery_soc

int16 drone_priority #

int64 ETA_next_WP #

uint32 drone_id #
